docs/linuxcredstores.md

```diff
@@ -0,0 +1,109 @@
+# Credential stores on Linux
```

Comment:
Please add links to this document in the docs for `GCM_CREDENTIAL_STORE` and `credential.credentialStore` for easier discovery.

Reply:
Done